app/repositories/application/GeneralApplicationRepository.scala

```diff
@@ -386,35 +388,53 @@ class GeneralApplicationMongoRepository(timeZoneService: TimeZoneService)(implic
       BSONDocument("applicationStatus" -> BSONDocument("$ne" -> "WITHDRAWN"))
     )))
 
-  override def overallReport(frameworkId: String): Future[List[Report]] =
+  override def overallReport(frameworkId: String): Future[List[CandidateProgressReport]] =
```

Comment:
If the report is named "Candidate Progress Report" I think is a good idea to rename the method to something like candidateProgressReport.